conf/prod.routes

```diff
@@ -1,5 +1,6 @@
 # Add all the application routes to the app.routes file
-->         /                     app.Routes
+->         /v1                   app_v1.Routes
+->         /vP1/sandbox          sandbox_vP1.Routes
```

Comment:
We should maintain consistency with naming of route files as [discussed in another pr](https://github.com/hmrc/individuals-employments-api/pull/2#pullrequestreview-59795844).

Reply:
👍